CHANGES.md

```diff
@@ -1,16 +1,18 @@
 ## 1.3.5 2018-01-17
 
+This release focuses on improvements and ixes for Namerd and the Linkerd Admin UI, and features a number of community contributions!
```

Comment:
I'd say something like "This release focuses on quality, and on improving the debugging process. It includes improvements and fixes for Linkerd's Kubernetes support, administrative UI, and Namerd control plane. It officially graduates HTTP/2 support out of experimental, and also features a number of community contributions!"